### DISCUSSION TITLE
Create meaningful and useful wiki home page

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#58] [General]
The discussion is moved from the [related issue](https://github.com/asastats/channel/issues/50).

-----

https://github.com/asastats/channel/wiki

Right now pages are just listed, we should create a starting point for something that is described in Channel's README as "A collaborated work on the official ASA Stats documents should take place in the [wiki](https://github.com/asastats/channel/wiki).".

**Comment by wxivd** [comment]
A collaboration and documentation repository for ASA Stats project

### Knowledge base

- Learn [About](https://www.asastats.com/about/) ASA Stats project and [FAQs](https://www.asastats.com/faq/).
- Learn about [ASA Stats Goals](https://github.com/asastats/channel/wiki/Goals).
- Learn about ASA Stats [DAO governors](https://github.com/asastats/channel/wiki/Governors) and [Governor seat update candidates](https://github.com/asastats/channel/wiki/GovernorCandidates).
- Learn about [Airdrop qualifiers](https://github.com/asastats/channel/wiki/AirdropQualifiers).

### Project Development
- Refer to [Roadmap](https://github.com/asastats/docs/blob/main/roadmap.md) for information on progress achieved. 
- [Dependent feature requests](https://github.com/asastats/channel/wiki/DependentFeatures) contains user's feature requests which require software functionality yet to be developed and implemented in ASA Stats. These include: [User settings](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserSettings), [User widgets](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserWidgets), [Mockfolio](https://github.com/asastats/channel/wiki/FeaturesDependentOnMockfolio), [Marketing campaign](https://github.com/asastats/channel/wiki/FeaturesDependentOnMarketingCampaign), [Historic data](https://github.com/asastats/channel/wiki/FeaturesDependentOnHistoricData), [QA testing](https://github.com/asastats/channel/wiki/FeaturesDependentOnQAtesting), [Advance bundles](https://github.com/asastats/channel/wiki/FeaturesDependentOnAdvanceBundles).

### Edit an use ASA Stats Channel

- Learn how to navigate the [ASA Stats Channel](https://github.com/asastats/channel).
- Learn how [Discussions](https://github.com/asastats/channel/wiki/Discussions) works.
- Learn the [rules and best practices for adding content and labels to Channel ](https://github.com/asastats/channel/discussions/3).

### Get help and contribute

- Cannot find the answer to a particular question? Ask in [discussions](https://github.com/asastats/channel/discussions).
- Report wrong software behaviour or a feature proposal in the [issues](https://github.com/asastats/channel/issues).
- Learn about [The ASA Stats Anti-shill Philosophy](https://github.com/asastats/channel/wiki/AntiShillPhilosophy).

Some content may be omitted (e.g, about, faqs, roadmap) to focus content on Github Wiki but I would rather keep them.

**Reply by ipaleka** [reply]
Rather a **rule** than a best practice, the wiki first/home page simply has to focus only on the wiki pages themselves without any other information. Take a look at the right-side panel for example, it brings all the pages created.

We need the start page to represent the wiki pages, while the explanation for the whole repository goes to the repository's root [README.md](https://github.com/asastats/channel).

**Reply by wxivd** [reply]
> We need the start page to represent the wiki pages, while the explanation for the whole repository goes to the repository's root [README.md](https://github.com/asastats/channel).

### Explanation for Wiki repository
The following is proposed to update [README.md](https://github.com/asastats/channel). 

**Welcome to ASA Stats official communication channel**

Learn about ASA Stats project development and how to make changes, contributions, or suggestions in [Wiki](https://github.com/asastats/channel/wiki), A collaboration and documentation repository for ASA Stats project.

Every [ASA Stats discussion](https://github.com/asastats/channel/wiki/Discussions) besides [DAO discussions](https://github.com/asastats/dao-discussion/discussions) should take place in [discussions](https://github.com/asastats/channel/discussions).

An issue for every bug report and a feature request should be created in the [issues](https://github.com/asastats/channel/issues).

There should be no offtopic and/or casual communication in the Channel, please use the ASA Stats Community channels for such a purpose: the ASA Stats [Discord server](https://discord.gg/Vjx7w7pAC7), [subreddit](https://www.reddit.com/r/asastats/), and [Twitter](https://twitter.com/asastatscom).

### Wiki Home
> focus only on the wiki pages themselves without any other information

Welcome ASA Stats Wiki


**Getting started**
- Learn about [ASA Stats Goals](https://github.com/asastats/channel/wiki/Goals).
- Learn about ASA Stats [DAO governors](https://github.com/asastats/channel/wiki/Governors) and [Governor seat update candidates](https://github.com/asastats/channel/wiki/GovernorCandidates).

**Project Development**
- [Dependent feature requests](https://github.com/asastats/channel/wiki/DependentFeatures) contains user's feature requests which require software functionality yet to be developed and implemented in ASA Stats. These include: These include: [User settings](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserSettings), [User widgets](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserWidgets), [Mockfolio](https://github.com/asastats/channel/wiki/FeaturesDependentOnMockfolio), [Marketing campaign](https://github.com/asastats/channel/wiki/FeaturesDependentOnMarketingCampaign), [Historic data](https://github.com/asastats/channel/wiki/FeaturesDependentOnHistoricData), [QA testing](https://github.com/asastats/channel/wiki/FeaturesDependentOnQAtesting), [Advance bundles](https://github.com/asastats/channel/wiki/FeaturesDependentOnAdvanceBundles).

**Edit and use ASA Stats Channel**

- Learn how to navigate the [ASA Stats Channel](https://github.com/asastats/channel).
- Learn how ASA Stats [Discussions](https://github.com/asastats/channel/wiki/Discussions) work.
- Learn the [rules](https://github.com/asastats/channel/discussions/3) for adding content and labels to Channel.

**Get help and contribute**

- Cannot find the answer to a particular question? Ask in [discussions](https://github.com/asastats/channel/discussions).
- Report wrong software behaviour or a feature proposal in the [issues](https://github.com/asastats/channel/issues).
- Learn about [The ASA Stats Anti-shill Philosophy](https://github.com/asastats/channel/wiki/AntiShillPhilosophy).
- Learn about [Airdrop qualifiers](https://github.com/asastats/channel/wiki/AirdropQualifiers).

### Add a custom sidebar
This would collapse pages on the right side and help organise featured pages 
<img width="123" alt="Screenshot 2022-05-06 at 12 37 16" src="https://user-images.githubusercontent.com/95645978/167125912-5662e7b7-869e-4947-8a1f-3c70de233b34.png">


_Sidebar

**Wiki**
- [Home](https://github.com/asastats/channel/wiki)
- [ASA Stats Goals](https://github.com/asastats/channel/wiki/Goals)

**DAO**
- [Governors](https://github.com/asastats/channel/wiki/Governors)
- [Candidates](https://github.com/asastats/channel/wiki/GovernorCandidates)

**Dependent feature requests**
- [User settings](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserSettings)
- [User widgets](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserWidgets)
- [Mockfolio](https://github.com/asastats/channel/wiki/FeaturesDependentOnMockfolio)
- [Marketing campaign](https://github.com/asastats/channel/wiki/FeaturesDependentOnMarketingCampaign)
- [Historic data](https://github.com/asastats/channel/wiki/FeaturesDependentOnHistoricData)
- [QA testing](https://github.com/asastats/channel/wiki/FeaturesDependentOnQAtesting)
- [Advance bundles](https://github.com/asastats/channel/wiki/FeaturesDependentOnAdvanceBundles)

**Reply by ipaleka** [reply]
This is closer to what we need, but we need to emphasize some other things.

First of all, we're about to design that starting wiki page right now and our goal has to be that it (kind of) never changes again. And again, think of wiki platform as a sandbox, there's nothing else in the world.

For example, this:

> Learn how to navigate the [ASA Stats Channel](https://github.com/asastats/channel).

There are two things we should do with this, At first, this remark from above, there should be no link that goes outside of the wiki platform. And the second thing is that ASA Stats discussion in this regard is just a step to a product, a discussion by itself is never a product!

**An empty wiki page is a feature, not a bug!**

So in this case we should do the following:

Learn how to navigate the [ASA Stats Channel](https://github.com/asastats/channel/wiki/ChannelHowto).

Once the related discussion is over we should create this wiki page from the outcomes of that discussion.

So, there's no room for something like this one:

> Cannot find the answer to a particular question? Ask in [discussions](https://github.com/asastats/channel/discussions).

That link can only exist on a wiki page that is entitled and explains something like "How to use ASA Stats Channel" and the starting wiki page can only link to such a page.

**Comment by wxivd** [comment]
### Wiki Home

Welcome ASA Stats Wiki

**Getting started**

- Learn about [ASA Stats Goals](https://github.com/asastats/channel/wiki/Goals).
- Learn about ASA Stats [DAO governors](https://github.com/asastats/channel/wiki/Governors) and [Governor seat update candidates](https://github.com/asastats/channel/wiki/GovernorCandidates).

**Project Development**

- [Dependent feature requests](https://github.com/asastats/channel/wiki/DependentFeatures) contains user's feature requests which require software functionality yet to be developed and implemented in ASA Stats. These include: These include: [User settings](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserSettings), [User widgets](https://github.com/asastats/channel/wiki/FeaturesDependentOnUserWidgets), [Mockfolio](https://github.com/asastats/channel/wiki/FeaturesDependentOnMockfolio), [Marketing campaign](https://github.com/asastats/channel/wiki/FeaturesDependentOnMarketingCampaign), [Historic data](https://github.com/asastats/channel/wiki/FeaturesDependentOnHistoricData), [QA testing](https://github.com/asastats/channel/wiki/FeaturesDependentOnQAtesting), [Advance bundles](https://github.com/asastats/channel/wiki/FeaturesDependentOnAdvanceBundles).

**Community rewards**
- Learn how to earn community reward (will propose page content if you find the entry suitable).
- Learn about [Airdrop qualifiers](https://github.com/asastats/channel/wiki/AirdropQualifiers).

**Edit and use ASA Stats Channel**
- Learn how to use ASA Stats Channel (proposed page content follows)
- Learn about [The ASA Stats Anti-shill Philosophy](https://github.com/asastats/channel/wiki/AntiShillPhilosophy).

### How to use ASA Stats Channel: Proposed page content

**Discussions**
- Learn how ASA Stats [Discussions](https://github.com/asastats/channel/wiki/Discussions) work.

**Feature requests and Bug reports**

If an issue directly expects a discussion about its topic, then a [Channel discussion](https://github.com/asastats/channel/discussions) should be created instead of an issue - an issue should then be created after that discussion outcomes with a solution or a path to it.

- An issue for every bug report and a feature request should be created in the [issues](https://github.com/asastats/channel/issues).
- Learn the [rules](https://github.com/asastats/channel/discussions/3) for adding content and labels to Channel.
- Learn about [Dependent feature requests](https://github.com/asastats/channel/wiki/DependentFeatures) which contain user's feature requests which require software functionality yet to be developed and implemented in ASA Stats

When somebody creates a request that is a dependable feature request then the following steps should be followed:

1. Create an entry in a related subpage of the [Dependent feature requests](https://github.com/asastats/channel/wiki/DependentFeatures).
A link to the issue inside that entry is mandatory (rewards-wise), while our good practice so far has been to either copy/paste the description or formulate the issue by your own words (clicking and reverting back to just getting to know what this is about should be avoided).
2. Add a comment with a link to that entry. The following [comment](https://github.com/asastats/channel/wiki/FeaturesDependentOnHistoricData#average-apy-for-staking) is a good example.
3. Label the issue with "wontfix".
4. Close the issue.

**Business proposal/opportunity**
If you have a business proposal, a suggestion, or a partnership possibility, we would love to hear from you in [discussions](https://github.com/asastats/channel/discussions).

I will propose custom sidebar update once we reach a consensus about the Wiki home page
